packages/api/src/store/asset-table.ts

```diff
@@ -43,6 +44,17 @@ export default class AssetTable extends Table<WithID<Asset>> {
   }
 
   async getByIpfsCid(cid: string, user?: User): Promise<WithID<Asset>> {
```

Comment:
Please check other uses of this function and make sure they still make sense (e.g. not making a separate redundant query for the source URL as well). Always a good practice when changing the behavior of a function, and this one specifically I **think** was used from the asset controller as well.